### PR TITLE
Add tap gse help doc

### PR DESCRIPTION
**Pull request by jon-perreault**
### What does this Pull Request accomplish?

Adds the GSE Tap Help reference document.

### Why should this Pull Request be merged?

Adds new documentation.

### What testing has been done?

Reviewed on https://jon-perreault.github.io/automotive-camera-module-reference/

### CODE REVIEW

File: docs/reference/gettingstartedexample/gse-tap-help.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+- **CSI-2 Data Source** - Selects the CSI-2 data source type. "Corrected" returns the received data after correcting for single-bit transmission errors. "Raw" returns the received data as is. 
+- **Deserializer (Input) Configuration Script** - Full path to a script file used to configure the deserializer.
+  #### Table of PXIe-148X Deserializer Tap Scripts
+
+  | **Interface Module**                                 | **Configuration Script**                              |
+  |------------------------------------------------------|-------------------------------------------------------|
+  | PXIe-1486 (4 In 4 Out - 953/954 SerDes)              | Host\\Scripts\\DS90UB954\\Tap\\Des_Tap.py             |
+  | PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes)          | Host\\Scripts\\MAX9296A\\Tap\\RAW12_ID01_Des_Tap.cpp  |
+  | PXIe-1487 (4 In 4 Out - 96717/96716A SerDes - KU11P) | Host\\Scripts\\MAX96716A\\Tap\\RAW12_ID12_Des_Tap.cpp |
+  | PXIe-1488 (4 In 4 Out - 971/9702 SerDes - KU11P)     | Host\\Scripts\\DS90UB9702\\Tap\\Des_Tap.py            |
```
Comment (hernstrom) [inline docs/reference/gettingstartedexample/gse-tap-help.md]: Remove ' - KU11P' for consistency

File: docs/reference/gettingstartedexample/gse-tap-help.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+- **Serializer (Output) Configuration Script** - Full path to a script file used to configure the serializer.
+  #### Table of PXIe-148X Serializer Tap Scripts
+
+  | **Interface Module**                                              | **Configuration Script**                                       |
+  |-------------------------------------------------------------------|----------------------------------------------------------------|
+  | PXIe-1486 (4 In 4 Out - 953/954 SerDes)                           | Host\\Scripts\\DS90UB953\\Tap\\Ser_Tap.py                      |
+  | PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes)                       | Host\\Scripts\\MAX9295A\\Tap\\RAW12_ID1_Ser_Tap.cpp            |
+  | PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes) Reverse Splitter Mode | Host\\Scripts\\MAX9295A\\Tap\\RAW12_ID0_ToRevSplit_Ser_Tap.cpp |
+  | PXIe-1487 (4 In 4 Out - 96717/96716A SerDes - KU11P)              | Host\\Scripts\\MAX96717\\Tap\\RAW12_ID2_Ser_Tap.cpp            |
+  | PXIe-1488 (4 In 4 Out - 971/9702 SerDes - KU11P)                  | Host\\Scripts\\DS90UB971\\Tap\\Ser_Tap.py                      |
```
Comment (hernstrom) [inline docs/reference/gettingstartedexample/gse-tap-help.md]: Remove ' - KU11P' for consistency

File: docs/reference/gettingstartedexample/gse-tap-help.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+- **Channel Configurations** - Array of active serial channel tap pairs and display parameters for each channel pair.
+    - **Serial Channel Tap Pair**
+        - **input** - String representing the active serial input channel in the tap pair in the format "SI" for Serial Input followed by the channel number (i.e. "SI0")
+        - **output** - String representing the active serial output channel in the tap pair in the format "SO" for Serial Output followed by the channel number (i.e. "SO0")
```
Comment (hernstrom) [inline docs/reference/gettingstartedexample/gse-tap-help.md:66]: Optional: could add note indicating numerical value of pairs must match

File: docs/reference/gettingstartedexample/gse-tap-help.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+  | None      | No power over coax provided.                                        |
+  | Internal  | Use internal power source to provide power over coax.               |
+  | Auxiliary | User power from the AUX POWER connector to provide power over coax. |
+
+- **Power Over Coax Sink** - The destination used to receive power over coax on active serial output channels.
+
+  | Value         | Description                                                |
+  |---------------|------------------------------------------------------------|
+  | **None**      | Power over coax disabled.                                  |
+  | **Auxiliary** | Route received power over coax to the AUX POWER connector. |
```
Comment (hernstrom) [inline docs/reference/gettingstartedexample/gse-tap-help.md]: 'None' and 'Auxiliary' appear to be accidentally **bolded**

**Review by hernstrom — APPROVED** [review]
Minor suggestions and cosmetic fixes. Approved for merge after making those small changes.

File: docs/reference/gettingstartedexample/gse-tap-help.md (hunk 1 above)
Comment (jon-perreault) [reply docs/reference/gettingstartedexample/gse-tap-help.md]: Done

File: docs/reference/gettingstartedexample/gse-tap-help.md (hunk 2 above)
Comment (jon-perreault) [reply docs/reference/gettingstartedexample/gse-tap-help.md]: Done

File: docs/reference/gettingstartedexample/gse-tap-help.md (hunk 3 above)
Comment (jon-perreault) [reply docs/reference/gettingstartedexample/gse-tap-help.md:66]: I added the note like you suggested.

File: docs/reference/gettingstartedexample/gse-tap-help.md (hunk 4 above)
Comment (jon-perreault) [reply docs/reference/gettingstartedexample/gse-tap-help.md]: This was copied and pasted from the Gen GSE Help document so I updated it there as well.

**Pull request merged**